src/services/LibraryService.ts

```diff
@@ -89,4 +97,36 @@ export class LibraryService implements Service<fhir4.Library> {
 
     return createLibraryPackageBundle(library[0], req.query['include-terminology'] ?? false);
   }
+
+  /**
+   * result of sending a POST request to:
+   * {BASE_URL}/4_0_1/Library/$submit or {BASE_URL}/4_0_1/Library/:id/$submit
+   * POSTs a new artifact in "draft" status. The operation results in an error if the artifact
+   * does not have status set to "draft."
+   */
+  async submit(args: RequestArgs, { req }: RequestCtx) {
+    logger.info(`${req.method} ${req.path}`);
+
+    const contentType: string | undefined = req.headers['content-type'];
+    checkContentTypeHeader(contentType);
```

Comment:
Out of scope for this task but just thought I'd note that it could be worth adding this function call to the other operations that send POST requests!